test/e2e/e2e_test.go

```diff
@@ -38,6 +40,128 @@ import (
 
 // +kubebuilder:docs-gen:collapse=Imports
 
+var _ = ginkgo.Describe("Kueue visibility server", func() {
+	const defaultFlavor = "default-flavor"
+	var (
+		defaultRF    *kueue.ResourceFlavor
+		localQueue   *kueue.LocalQueue
+		clusterQueue *kueue.ClusterQueue
+		ns           *corev1.Namespace
+		sampleJob1   *batchv1.Job
+		sampleJob2   *batchv1.Job
+	)
+
+	ginkgo.BeforeEach(func() {
+		ns = &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "e2e-",
+			},
+		}
+		gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
+	})
+	ginkgo.AfterEach(func() {
+		gomega.Expect(util.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
+	})
+
+	ginkgo.When("Creating two Jobs with max capacity", func() {
```

Comment:
```suggestion
	ginkgo.When("There are pending workloads due to max capacity by the admitted job", func() {
```
Maybe it better captures the scenario?